### DISCUSSION TITLE
Cannot open the stats tab

### DISCUSSION THREAD

**Discussion by rogegla** [YOURLS/YOURLS#3598] [Q&A]
Hello!

I cannot open the stats tab at all for some reason.

I can view the admin panel and settings and stuff fine, but when I click the stats button
the browser shows me the folder and files that are in the folder where yourls is.

**Comment by dgw** [comment]
So, do your shortlinks redirect properly? Because the same mechanism (what the webserver is told to do for paths that don't exist on the filesystem) routes both redirections and stats.

**Reply by rogegla** [reply]
Redirects work fine, but stats shows the index / of the folder. Does it make any difference that i use vps and not web hosting?

**Reply by dgw** [reply]
It does not matter, if the VPS is set up correctly.

Visiting `https://short.tld/keyword` works, but `https://short.tld/keyword+` doesn't? I think we need some details about the environment: HTTP server, OS, version numbers (including PHP), that sort of thing.

Check logs and do the other suggestions in [Troubleshooting: First Steps](https://yourls.org/docs/guide/troubleshooting/first-steps) if you haven't yet.
